### DISCUSSION TITLE
use it on local tigergraph

### DISCUSSION THREAD

**Discussion by jwx123** [graphistry/graph-app-kit#85] [Q&A]
Can i use it connect the local tigergraph and display the graph create by myself

**Comment by lmeyerov** [comment]
Yep!

- You can use TigerGraph's python dataframes bindings to pick the data to show to users: https://github.com/graphistry/graph-app-kit/blob/master/src/python/views/demo_tigergraph_fraud/__init__.py

- We also have experimental direct Python bindings so you can go directly from query to dataframe & viz: https://github.com/graphistry/pygraphistry/blob/master/demos/demos_databases_apis/tigergraph/tigergraph_pygraphistry_bindings.ipynb

The tigergraph instance can be your local one, and graphistry can be hub or a self-hosted one (`graphistry.register(...)`)
